MetaMask/metamask-extension
# Call Wyre’s API via our backend to generate Wyre’s Checkout URL

## Explanation
Call Wyre’s API via our backend to generate Wyre’s Checkout URL. It's a strongly recommended way how to create their Checkout URL, because it has some additional query params that we don't currently have.

## Manual testing steps
  - Open the MetaMask browser extension
  - Click on `Buy` on the main page
  - Click on `Continue to Wyre` on the `Deposit Ether` Page
  - Check that a new tab is opened with Wyre's Checkout URL, has prefilled ETH address and there is a query param called `reservation`. If it's not there, it means a fallback URL was used instead. 

## Screenshots
`Deposit Ether` Page:
![image](https://user-images.githubusercontent.com/80175477/123408870-cc628800-d5ad-11eb-96fb-e55a62740c11.png)
